internal/controller/numaflowcontroller/numaflowcontroller_controller.go

```diff
@@ -454,29 +358,54 @@ func determineTargetObjects(
 	if err != nil {
 		return nil, fmt.Errorf("can not parse file data, err: %w", err)
 	}
-	manifestsWithOwnership, err := applyOwnershipToManifests(manifests, controller)
-	if err != nil {
-		return nil, fmt.Errorf("failed to apply ownership reference, %w", err)
-	}
```

Comment:
we no longer apply OwnerReference since gitops-engine assumes there's no OwnerReferences